docker-images/base-alpine-python36/harness.py

```diff
@@ -15,15 +15,16 @@
 from urllib3.util import parse_url
 
 
-def get_logger(name):
+def get_logger(name, id_name, id_val):
     ch = logging.StreamHandler(sys.stdout)
     log_level_name = os.environ['LOG_LEVEL'] if 'LOG_LEVEL' in os.environ else 'DEBUG'
     log_level = getattr(logging, log_level_name.upper())
     ch.setLevel(log_level)
-    formatter = logging.Formatter('%(asctime)s %(levelname)s %(name)s %(message)s',
-                                  datefmt="%Y-%m-%dT%H:%M:%S%z")
+    formatter = logging.Formatter(f'cor_id:{id_name}:{id_val} timestamp:%(asctime)s level:%(levelname)s' +
```

Comment:
This format seems a little verbose to me. I would drop the `timestamp:`, `level:`, `name:` and `message:` prefixes.

Developers can generally recognize these fields if they're in a standard format (see https://en.wikipedia.org/wiki/Common_Log_Format or check out standard formats for SLF4J). Readability (having things fit comfortably in a terminal or browser) and data size probably outweigh being super explicit.

Devs will still be able to easily parse these messages with tools such as `awk`, `grep`, `cut`, and `jq`.

Reply:
nit: Also, maybe going general to specific would help. Something like:

`[iso8601_timestamp] [module] [thread_id] [level] [correlation_id] [message]`

The rationale is

`time > code_module > thread > message_type > entity > message`

Reply:
Agreed on both counts